src/components/Header/Header.module.css

```
@@ -1,20 +1,126 @@
 .header {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-    position: fixed;
-    width: 100%;
-    top: 0;
-    left: 0;
-    z-index: 1000;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 48px 24px;
```

Having the padding-bottom is currently adding to match space between header and about page first section and contact page since it has already been added, but we will need to update the home-first section to reflect the same
```suggestion
  padding: 48px 24px 0;
```

---

I will take that up in a separate PR